app/views/admin/infobase/_today_menu.html.haml

```diff
@@ -0,0 +1,53 @@
+%div
```

Comment:
This is awful file and don't even want to look at it. Please refactor based on my previous comments and I will take a look